README.md

```diff
@@ -1,2 +0,0 @@
-# Week11-hangman
-Homework: a node-based hangman game with recursive functions.
```

Comment:
couple of things:

1. always have `node_modules` inside of a `.gitignore` file. otherwise git is gonna commit all of it into your repo which is why you ended up with 1,615 files in this repo.

2. most times you will pull-request your branch to master (which is what you did here). however, in this case, we wanted to compare *from* master *to* `blankBranch` (just switch the base and branch when opening a PR) and that would show your code in green (addition vs deletion).